tests/network/docker-compose.yml

```
@@ -73,7 +73,7 @@ services:
       OCRD_NETWORK_LOGS_ROOT_DIR: /ocrd-data/ocrd_network_logs
       OCRD_NETWORK_SOCKETS_ROOT_DIR: /ocrd-data/ocrd_network_sockets
     healthcheck:
-      test: curl -f ${OCRD_PS_URL}/
+      test: curl -f http://localhost:${OCRD_PS_PORT}/
```

```suggestion
      test: curl -f http://localhost:8000/
```

we are on the PS container here, so the **internal port** must be used

---

Oh, right...